workflow/rocoto/tasks.py

```diff
@@ -223,7 +223,16 @@ def get_resource(self, task_name):
             else:
                 native += ':shared'
         elif scheduler in ['slurm']:
-            native = '--export=NONE'
+           #The PW_CSP is a AWS (CSPs parameter), if it is on CSPs, we need 'native' defined
+           #as below. Or, it won't run, but with an error:
+           #"ufs_model.x: error while loading shared libraries: libiomp5.so: cannot open shared object file: No such file or directory"
+           #Even the library path is clearly in LD_LIBRARY_PATH, or load exactly the modules when build ufs_model.x
+           #TODO: find a mechanism to provide native scheduler information.
+            pw_csp = os.environ.get('PW_CSP', 'unknown')
+            if ( pw_csp in ['aws', 'azure', 'google'] ):
+                native = '--export=ALL --exclusive'
+            else:
+                native = '--export=NONE'
```

Comment:
This should be unnecessary, if 
`export is_exclusive="YES"` is set in `config.resources.AWSPW`

Reply:
made this changes. and tested, it worked.
Thanks.